src/features/generic/time-utils.ts

```diff
@@ -56,6 +56,37 @@ namespace TimeUtilities {
       displayInternal: displayInternal,
     }
   }
-};
+
+  /**
+   * Changes project times to percents and makes over- and undertime messages
+   * 
+   * @param user data from timebank
+   * @returns project time percents and over- and undertime messages
+   */
+  export const calculateWorkedTimeAndBillableHours = (user: TimeEntryTotalDto | DailyCombinedData) => {
+    const { projectTime, expected,total } = user;
+
+    const billableHours = (projectTime/expected * 100).toFixed(1);
+    const billableHoursWithPercentage = `${billableHours}%`;
+
+    const overtime = TimeUtilities.timeConversion(total)
+    const undertime = TimeUtilities.timeConversion(total * -1)
+
+    let under_over_message = "";
+    if(total < 0){
+      under_over_message = "Undertime: " + undertime;
+    }else if(total > 0){
+      under_over_message = "Overtime: " + overtime;
+    }else{
+      under_over_message = "No overtime, you worked the expected time."
+    }
```

Comment:
I'd change this so that the message variable has the "No overtime, you worked the expected time." as initial value. Then separate first two checks into own separate if checks. Also use string literal syntax

Reply:
Done